src/vueWrapper.ts

```diff
@@ -38,7 +50,124 @@ export class VueWrapper<T extends ComponentPublicInstance> {
     return this.vm.$el.parentElement
   }
 
+  /**
+   * Get all the target elements defined in the teleportTarget mount option.
+   *
+   * @private
+   *
+   * @return {Element[]|undefined}
+   */
+  private getTeleportElements(): Element[] | undefined | never {
```

Comment:
`| never` is necessary here?

Reply:
Maybe we could avoid returning undefined, and just return an empty array if there are no elements? It would avoid TS users to have to add a non-null assertion when writing a test.